composer.json

```diff
@@ -12,19 +12,25 @@
   "license": "BSD-3-Clause",
   "require": {
     "php": "^7.2",
+    "ext-curl": "*",
+    "ext-filter": "*",
+    "ext-gd": "*",
     "ext-intl": "*",
     "ext-json": "*",
+    "ext-openssl": "*",
     "ext-pdo": "*",
     "guzzlehttp/guzzle": "^6.3",
     "guzzlehttp/oauth-subscriber": "^0.3.0",
     "michelf/php-markdown": "^1.8",
     "pimple/pimple": "^3.2",
+    "psr/container": "^1.0.0",
```

Comment:
These dependencies are added because they are required - not by `maglnet/composer-require-checker`, but by this application.

Reply:
So this package ensures anything we're using in our code that isn't explicitly stated in our composer.json, gets added there, and a situation where that is currently happening is we're using `psr/container` but not from our own composer.json? From another dependency that also uses it?